spec/helpers/formatting_helper_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe FormattingHelper, type: :helper do
+  include Devise::Test::ControllerHelpers
```

Comment:
This seemed to be necessary because the helper would eventually call into a users emoji settings (I have lost the stacktrace but I can dig it up if necessary)